inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -2989,94 +3032,233 @@ TEST_P(activation_quantize_i8, basic) {
                       data("out_low", get_mem(get_single_element_layout(p), -127, 0)),
                       data("out_high", get_mem(get_single_element_layout(p), 0, 127)),
                       quantize("quant", "act", "in_low", "in_high", "out_low", "out_high", 255, data_types::i8),
-                      reorder("reorder_bfyx", "quant", format::bfyx, data_types::f32));
+                      reorder("reorder_bfyx", "quant", p.default_format, data_types::f32));
 
-    tolerance = 1.f;
+    tolerance = 1.0f;
     execute(p);
 }
 
 INSTANTIATE_TEST_CASE_P(
     fusings_gpu,
     activation_quantize_i8,
     ::testing::ValuesIn(std::vector<activation_test_params>{
-        activation_test_params{CASE_ACTIVATION_F32_1, 2, 3},
-        activation_test_params{CASE_ACTIVATION_F32_2, 2, 3},
-        activation_test_params{CASE_ACTIVATION_F32_3, 2, 3},
-        activation_test_params{CASE_ACTIVATION_F32_4, 2, 3},
-        activation_test_params{CASE_ACTIVATION_F32_5, 2, 3},
-        activation_test_params{CASE_ACTIVATION_F32_6, 2, 3},
-        activation_test_params{CASE_ACTIVATION_3D_F32_1, 2, 3},
-        activation_test_params{CASE_ACTIVATION_3D_F32_2, 2, 3},
-        activation_test_params{CASE_ACTIVATION_3D_F32_3, 2, 3}
+        // InputDataType = FP32
+        activation_test_params{CASE_ACTIVATION_F32_0, 2, 3, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_F32_1, 2, 3, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_F32_2, 2, 3, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_F32_3, 2, 3, "activation_opt"},
+
+        activation_test_params{CASE_ACTIVATION_3D_F32_0, 2, 3, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_1, 2, 3, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_2, 2, 3, "activation_opt"},
+        activation_test_params{CASE_ACTIVATION_3D_F32_3, 2, 3, "activation_opt"},
```

Comment:
Is there a test case when activation_opt kernel is used and X % 4 != 0?

Reply:
Added.